pkgs/applications/misc/hunter/default.nix

```
@@ -0,0 +1,52 @@
+{ lib
+, stdenv
+, pkg-config
+, rustPlatform
+, fetchFromGitHub
+, makeWrapper
+, glib
+, gst_all_1
+, CoreServices
+, IOKit
+, Security
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "hunter";
+  version = "1.3.5";
+
+  src = fetchFromGitHub {
+    owner = "06kellyjac";
+    repo = "hunter";
+    rev = "2484f0db580bed1972fd5000e1e949a4082d2f01";
+    sha256 = "sha256-oSuwM6cxEw4ybiwoYX6A/aqiU6NVu9cLLONalUHuE1A=";
+  };
+
+  RUSTC_BOOTSTRAP = 1;
+
+  nativeBuildInputs = [ makeWrapper pkg-config ];
+  buildInputs = [
+    glib
+    gst_all_1.gstreamer
+    gst_all_1.gst-plugins-base
+    gst_all_1.gst-plugins-good
+    gst_all_1.gst-plugins-ugly
+    gst_all_1.gst-plugins-bad
+  ] ++ lib.optionals stdenv.isDarwin [ IOKit ];
```

You forgot to add the packages to buildInputs.

```suggestion
  ] ++ lib.optionals stdenv.isDarwin [ CoreServices IOKit Security ];
```